android/app/src/main/java/app/organicmaps/DownloadResourcesLegacyActivity.java

```diff
@@ -44,15 +53,6 @@ public class DownloadResourcesLegacyActivity extends BaseMwmFragmentActivity
 {
   private static final String TAG = DownloadResourcesLegacyActivity.class.getSimpleName();
 
-  // Error codes, should match the same codes in JNI
```

Comment:
Can this refactoring be merged separately?

Reply:
This refactoring is for this PR - move to sdk package
No reason to do it separately